src/metabase/search/scoring.clj

```diff
@@ -127,30 +128,50 @@
                           [(str/lower-case (name model)) i])
                         search-config/searchable-models)))
 
-(defn- score-with-match
+(defn- text-score-with-match
   [query-string result]
   (when (seq query-string)
-    (score-with match-based-scorers
-                (tokenize (normalize query-string))
-                result)))
+    (text-score-with match-based-scorers
+                     (tokenize (normalize query-string))
+                     result)))
+
+(defn- pinned-score
```

Comment:
Is collection_position a negative number that moves it to the front of a list and you then make it positive to boost the score? If so that might be a helpful little docstring or comment